include/tunnler/room_member.h

```diff
@@ -160,6 +166,7 @@ class RoomMember final {
     // cause an overflow in UDS::RecvBeaconBroadcastData.
     static const size_t MaxBeaconQueueSize = 25;
 
+    std::mutex chat_mutex;             ///< Mutex to protect access to the chat queue.
     std::deque<ChatEntry> chat_queue;    ///< List of all chat messages received since last PopChatEntries was called
```

Comment:
Do we really need this?

I was thinking we could just provide a callback and let the users of the library handle keeping a log of chat messages.